UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileShareService.cs

```diff
@@ -310,6 +314,96 @@ public async Task<string> SearchReadMeFilePath(string batchId,string correlation
             return filePath;
         }
 
+        public bool CreateZipFileForExchangeSet(string exchangeSetZipRootPath, string correlationId)
+        {
+            var zipName = $"{exchangeSetZipRootPath}.zip";
+            string path = Path.Combine(exchangeSetZipRootPath, zipName);
```

Comment:
Would the variable path be better names as filePath?

Reply:
Done.